etc/profile-m-z/steam.profile

```diff
@@ -12,6 +12,7 @@ noblacklist ${HOME}/.config/MangoHud
 noblacklist ${HOME}/.config/ModTheSpire
 noblacklist ${HOME}/.config/RogueLegacy
 noblacklist ${HOME}/.config/RogueLegacyStorageContainer
+noblacklist ${HOME}/.factorio
```

Comment:
Please add the counterpart `blacklist ${HOME}/.factorio` to `disable-programs.inc`. Noblacklisting otherwise won't be doing anything meaningful.

Reply:
Thanks for the feedback. Change added as requested.